website/server/libs/api-v3/pushNotifications.js

```diff
@@ -92,6 +96,25 @@ module.exports = function sendNotification (user, details = {}) {
             data: payload,
           });
         }
+
+        if (fcm) {
+          var message = {
+              to: pushDevice.regId,
+              data: payload,
+              notification: {
+                  title: details.title,
+                  body: details.message,
+              }
+          };
+
+          fcm.send(message, function(err, response){
+              if (err) {
+                  console.log(err, "Something has gone wrong!");
+              } else {
+                  console.log("Successfully sent with response: ", response);
```

Comment:
this should be removed, if the notification is sent correctly this isn't really necessary